pyproject.toml

```diff
@@ -5,6 +5,7 @@ readme = "README.md"
 requires-python = ">=3.12"
 dependencies = [
     "a2a-sdk>=0.3.0",
+    "fastapi[standard]",
```

Comment:
Thanks for adding this, but this should be done by using the  `a2a-sdk[http-server]` extra instead.